.github/workflows/CI.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           # compile packages and run GAP tests in 32 bit mode
           # it seems profiling is having trouble collecting the coverage data
           # here, so we use NO_COVERAGE=1
-          - os: ubuntu-latest
+          - os: ubuntu-18.04
```

Comment:
Actually perhaps we should run most  tests on 18.04 or an even old version (if available)? In the past I saw more issues where we broke things for old Linux versions (by using things "too new") than on new ones.